client/src/components/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+
+export default function Navbar() {
+  return (
+    <div className='nav'>
+          <Link to="/">
```

Comment:
@Benjamin-Ortiz 's request we will change this /exams for now so logo and pressing exams redirects to /exams, let me know if you want to include the logo.png just for dev purposes

or you can just leave 'LOGO' and just have it redirect to /exams as needed